sqlflow/client.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+import sys
+import logging
+
+import grpc
+import sqlflow_pb2
+import sqlflow_pb2_grpc
+import google.protobuf.wrappers_pb2 as wrapper
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--url", type=str, help="server url", action="store", required=True)
+
+#TODO(tonyyang-svail): move logging config elsewhere
+logging.basicConfig(format='%(asctime)s,%(msecs)d %(levelname)-8s [%(filename)s:%(lineno)d] %(message)s',
+    datefmt='%Y-%m-%d:%H:%M:%S',
+    stream=sys.stdout,
+    level=logging.DEBUG)
+
+
+class Client(object):
+    def __init__(self, server_url):
+        channel = grpc.insecure_channel(server_url)
+        self._stub = sqlflow_pb2_grpc.SQLFlowStub(channel)
+
+    def _decode_any(self, a):
+        if a.Is(wrapper.Int64Value.DESCRIPTOR):
```

Comment:
Need more robust way to decode all supported wrapper type

Reply:
| Protobuf Wrapper | Python |
|------------------|--------|
| BoolValue        | bool   |
| FloatValue       | float  |
| DoubleValue      | float  |
| Int32Value       | int    |
| Int64Value       | int    |

Reply:
Done.